Scripts/Ports/python3-cairosvg/vcpkg.json

```diff
@@ -6,8 +6,8 @@
   "dependencies": [
     {
       "name": "cairo",
-      "platform": "windows",
-      "default-features": false
+      "default-features": false,
+      "host": true
```

Comment:
`python3-cairosvg` in the project's manifest should probably be `"host": true` so that it will cascade properly. What you seem to be asking for here, in my mind, if you're on x64 macOS cross-compiling for ARM, x64 cairo and ARM python3.

Reply:
I think there is a bit of breakage going on in vcpkg that I've talked about with @dpogue that may be at play here. But - just to clarify - is Cairo only used for Resource.dat generation? I think it's only needed by the host.

What's actually needed in a cross compile (that might be broken right now):

- Client architecture Python
- Host architecture Python for build processes
- Host architecture Cairo
- Host architecture python3-cairo

(The above assumes that my understanding of Cairo as a build related tool is correct and not part of the client.)

Reply:
This is correct. We want for cairo to cascade to host arch by marking python3-cairosvg as host.

Reply:
I may be confused because I feel like this is the opposite of this comment here:
https://github.com/H-uru/Plasma/pull/1467#discussion_r1299233045

I think I'm confused on whats being asked for.

Reply:
In the root vcpkg.json:

```json
{
    "name": "python3-cairosvg,
    "host": true
}
```

In python3-cairosvg's json, `cairo` is a regular dependency.

Reply:
I think I understand what you're asking and I edited the last commit. Let me know if thats what you were thinking. I removed Cairo from the root manifest and am allowing python3-cairosvg to pull it through.

Reply:
I think you do need to remove the `"platform": "windows"` line in python3-cairosvg's config file though, otherwise it's going to skip installing cairo on macOS: https://github.com/H-uru/Plasma/blob/master/Scripts/Ports/python3-cairosvg/vcpkg.json#L9